{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -76,56 +95,68 @@ Ready to contribute? Here's how to set up ``{{ cookiecutter.project_slug }}`` fo
+
+  You can also run the hooks manually with::
 
     $ pre-commit run -a
 
+  If you want to skip the ``pre-commit`` hooks temporarily, you can pass the ``--no-verify`` flag to `$ git commit`.
+
 #. Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
-   Now you can make your changes locally.
+  Now you can make your changes locally.
 
-#. When you're done making changes, check that your changes pass ``black``, ``blackdoc``, ``flake8``, ``isort``, ``ruff``, and the tests, including testing other Python versions with tox::
+#. When you're done making changes, we **strongly** suggest running the tests in your environment or with the help of ``tox``::
 
-    $ black --check {{ cookiecutter.project_slug }} tests
-    $ isort --check {{ cookiecutter.project_slug }} tests
-    $ ruff {{ cookiecutter.project_slug }} tests
-    $ flake8 {{ cookiecutter.project_slug }} tests
-    $ blackdoc --check {{ cookiecutter.project_slug }} docs
     $ python -m pytest
+    # Or, to run multiple build tests
     $ tox
 
-   To get ``black``, ``blackdoc``, ``flake8``, ``isort``, ``ruff``, and tox, just pip install them into your virtualenv.
-
 #. Commit your changes and push your branch to GitHub::
 
     $ git add .
     $ git commit -m "Your detailed description of your changes."
     $ git push origin name-of-your-bugfix-or-feature
 
-#. If you are editing the docs, compile and open them with::
+  If ``pre-commit`` hooks fail, try re-committing your changes (or, if need be, you can skip them with `$ git commit --no-verify`).
 
+#. Submit a `Pull Request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`_ through the GitHub website.
+
+#. When pushing your changes to your branch on GitHub, the documentation will automatically be tested to reflect the changes in your Pull Request. This build process can take several minutes at times. If you are actively making changes that affect the documentation and wish to save time, you can compile and test your changes beforehand locally with::
+
+    # To generate the html and open it in your browser
     $ make docs
-    # or to simply generate the html
-    $ cd docs/
-    $ make html
+    # To uniquely generate the html
+    $ make autodoc
+    $ make -C docs html
+    # To simply test that the docs pass build checks
+    $ tox -e docs
+
+#. Once your Pull Request has been accepted and merged to the ``main`` branch, several automated workflows will be triggered:
 
-#. Submit a pull request through the GitHub website.
+    - The ``bump-version.yml`` workflow will automatically bump the patch version when pull requests are pushed to the ``main`` branch on GitHub. **It is not necessary to manually bump the version in your branch when merging (non-release) pull requests.**
```

Comment:
"not necessary" or "not recommended"? If I manually bump within my PR, will `bump-my-version` be intelligent enough to understand it and not bump a second time?
```suggestion
    - The ``bump-version.yml`` workflow will automatically bump the patch version when pull requests are pushed to the ``main`` branch on GitHub. **It is thus not necessary to manually bump the version in your branch when merging (non-release) pull requests.**
```

Reply:
It will not be intelligent enough to know (and coding that would be _incredibly_ annoying to do).

Reply:
```suggestion
    - The ``bump-version.yml`` workflow will automatically bump the patch version when pull requests are pushed to the ``main`` branch on GitHub. **It is not recommended to manually bump the version in your branch when merging (non-release) pull requests (this will cause the version to be bumped twice).**
```